test/modules/dynamic-imports.js

```diff
@@ -0,0 +1,19 @@
+var test = require('tape');
+require('../../src/ImageSequencer.js');
+
+var sequencer = ImageSequencer({ ui: false });
+var red = "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCAAQABADASIAAhEBAxEB/8QAFQABAQAAAAAAAAAAAAAAAAAAAAf/xAAUEAEAAAAAAAAAAAAAAAAAAAAA/8QAFQEBAQAAAAAAAAAAAAAAAAAABgj/xAAUEQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIRAxEAPwCdABykX//Z";
+
+test('Dynamically add a test Module', function(t){
+  sequencer.loadNewModule('test',{func: require('./testModule/Module'),info: require('./testModule/info')});
+  sequencer.loadImages('image1', red);
+  t.deepEqual(sequencer.modules['test'],[
+    require('./testModule/Module'),
+    require('./testModule/info')
+  ],"test module was succesfully imported");
+  sequencer.addSteps('invert');
+  sequencer.addSteps('test');
+  sequencer.addSteps('invert');
+  sequencer.run();
```

Comment:
Here, could we confirm that the input data is different after an invert, and same again after another invert?

Reply:
@jywarren we have those tests here, I think that this is just for testing if the import functinality is working correctly so we should not repeat these, what do you say?
https://github.com/publiclab/image-sequencer/blob/0c0147354e679fdc561b194ec8b98d0bfbefc1ac/test/modules/image-manip.js#L28
https://github.com/publiclab/image-sequencer/blob/0c0147354e679fdc561b194ec8b98d0bfbefc1ac/test/modules/image-manip.js#L40